Hush/Assets/Plugins/PyroParticles/Prefab/Script/FireProjectileScript.cs

```diff
@@ -98,12 +96,6 @@ public void HandleCollision(GameObject obj, Collision c)
                 ProjectileCollisionSound.Play();
             }
 
-            IKillable killable = c.gameObject.GetComponent<IKillable>();
-            if(killable != null)
-            {
-                killable.TakeDamage(Damage);
```

Comment:
why do we remove applying damage to the IKillable instance we collide with? I thought we had said to keep the damage on the attacks themselves? not too sure if i missed something

Reply:
Same as other comment